### DISCUSSION TITLE
Questions About PDF Compression Capabilities in PDFBox

### DISCUSSION THREAD

**Discussion by kishor-shinde-96** [community/community#144763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am exploring PDFBox for optimizing PDF files and have a few questions about its compression capabilities:

Does PDFBox support standard PDF compression methods, and what is typically considered "normal compression"?
Can we specify a compression range or level, such as setting compression to 80% or 90%?
Are all compression-related features available in the open-source version of PDFBox?

**Comment by queenofcorgis** [comment]
Hi @kishor-shinde-96 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, like [their website](https://pdfbox.apache.org/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
